communityrust.properties

```diff
@@ -1,11 +1,18 @@
 category=Languages
 description=Code Analyzer for Rust
 homepageUrl=https://github.com/elegoff/sonar-rust
-publicVersions=0.0.6
+
+publicVersions=0.0.7
 
 defaults.mavenGroupId=org.elegoff
 defaults.mavenArtifactId=community-rust-plugin
 
+0.0.7.description=Update with latest Rust grammer and refeshed list of Clippy list 
+0.0.7.sqVersions=[7.9,LATEST]
```

Comment:
Support for anything <8.9 is officially over. This range needs to start with 8.9